scripts/update-input.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
```

Comment:
Is this script used now?

Reply:
I forgot to use it in autorelease, will fix it now